plugin-name/includes/class-plugin-name-deactivator.php

```diff
@@ -22,14 +23,132 @@
 class Plugin_Name_Deactivator {
 
 	/**
-	 * Short Description. (use period)
+	 * The $_REQUEST during plugin activation.
 	 *
-	 * Long Description.
+	 * @since    1.0.0
+	 * @access   private
+	 * @var      array    $request    The $_REQUEST array during plugin activation.
+	 */
+	private static $request = array();
+
+	/**
+	 * The $_REQUEST['plugin'] during plugin activation.
+	 *
+	 * @since    1.0.0
+	 * @access   private
+	 * @var      string    $plugin    The $_REQUEST['plugin'] value during plugin activation.
+	 */
+	private static $plugin  = 'plugin-name/plugin-name.php';
+
+	/**
+	 * The $_REQUEST['action'] during plugin activation.
+	 *
+	 * @since    1.0.0
+	 * @access   private
+	 * @var      array    $action    The $_REQUEST[action] value during plugin activation.
+	 */
+	private static $action  = 'deactivate';
+
+	/**
+	 * Activate the plugin.
+	 *
+	 * Checks if the plugin was (safely) activated.
+	 * Place to add any custom action during plugin activation.
 	 *
 	 * @since    1.0.0
 	 */
 	public static function deactivate() {
 
+		if ( false === self::get_request()
+			|| false === self::validate_request( self::$plugin, self::$action )
+			|| false === self::check_caps()
+			|| ! check_admin_referer( 'deactivate-plugin_' . self::$request['plugin'] )
+		 ) {
```

Comment:
I think I found the cause here. Closing `)` had a space before it, instead of being aligned like it happens when you return it fresh from a line above it. Fixed and 🤞 it was that.